src/styles/utils/header.scss

```diff
@@ -0,0 +1,72 @@
+.head {
+  padding-left: 50px;
+  padding-right: 50px;
+  font-weight: 500;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &--flex {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+}
+
+a {
+  display: inline-block;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: v.$col-nav-link;
```

Comment:
Ensure that the variable `v.$col-nav-link` is defined in your variables file. If it's not defined, this will cause an error during compilation.